android/build.gradle

```diff
@@ -1,19 +1,19 @@
 
 buildscript {
-  repositories {
+    ext.kotlin_version = '1.5.30'
+    repositories {
     google()
     mavenCentral()
   }
 
   dependencies {
     classpath 'com.android.tools.build:gradle:4.2.2'
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
Just a heads up.
Should we use the camel case style like: `kotlinVersion`?

Not sure if `kotlin_version` is whether sort of the android studio default or not, but I see lots of example use this one.
But in react native realm, I saw some popular repos use camel case style.

ex:

https://github.com/software-mansion/react-native-screens/blob/master/android/build.gradle#L12 
https://github.com/react-native-webview/react-native-webview/blob/master/android/build.gradle#L17
https://github.com/Nozbe/WatermelonDB/blob/master/native/android/build.gradle#L38

I'm worried I have to use both `kotlin_version` and `kotlinVersion` in my gradle ext eventually :fearful: 
Not a big deal though, thanks for the great job. :+1:

Reply:
Thanks for issuing! Looks like we need to fix to `$kotlinVersion`.